src/clogconfig.rs

```diff
@@ -4,6 +4,7 @@ use std::path::Path;
 use std::fmt::Display;
 use std::env;
 use std::collections::HashMap;
+use std::ascii::AsciiExt;
```

Comment:
Just saw this, it's not necessary since the `arg_enum!{}` macro also expands to include this.